### ISSUE TITLE
Changing context path on Jetty may cause EoF exception at BaseTagHrefFilter

### ISSUE THREAD

**Issue by tadayosi** [hawtio/hawtio#2621]
Trying to change the context path: `/hawtio` -> `/console` causes this error on ActiveMQ Artemis:
```
org.eclipse.jetty.io.EofException: Closed
	at org.eclipse.jetty.server.HttpOutput.checkWritable(HttpOutput.java:719)
	at org.eclipse.jetty.server.HttpOutput.write(HttpOutput.java:749)
	at java.io.OutputStream.write(OutputStream.java:75)
	at io.hawt.web.filters.BaseTagHrefFilter.doFilter(BaseTagHrefFilter.java:79)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.auth.LoginRedirectFilter.doFilter(LoginRedirectFilter.java:57)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.filters.HttpHeaderFilter.doFilter(HttpHeaderFilter.java:43)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1604)
	at io.hawt.web.auth.SessionExpiryFilter.process(SessionExpiryFilter.java:97)
	at io.hawt.web.auth.SessionExpiryFilter.doFilter(SessionExpiryFilter.java:60)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1596)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:545)
```

**Comment by tadayosi** [comment]
Could be due to this Jetty bug: https://github.com/eclipse/jetty.project/issues/4533 and if so it should be fixed with Jetty 9.4.27. Let's check it.

**Comment by tadayosi** [comment]
Tested with Jetty 9.4.27.v20200227 and indeed it's fixed with the version. Closing.
